### DISCUSSION TITLE
No documents found

### DISCUSSION THREAD

**Discussion by ChettakattuA** [PromtEngineer/localGPT#172] [Q&A]
File "C:\Users\ChettakattuA\Documents\AI\LocalGPT\localGPT\run_localGPT_API.py", line 48, in <module>
    raise FileNotFoundError(
FileNotFoundError: No files were found inside SOURCE_DOCUMENTS, please put a starter file inside before starting the API!

this the error i get when run API but source document directory does have files

**Comment by huycke** [comment]
I'm also curious about this issue. I'm trying to change the source_documents directory location, and while it worked for the ingest.py and the run_localGPT.py files it doesn't seem to work when I'm trying to run the UI.

**Comment by czw-czw-czw** [comment]
Do you solve this problem?

**Comment by huycke** [comment]
I don't have the issue anymore, but can't remember what it was that fixed it.
